config/manifests/bases/mtk-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  name: mtk-operator.v0.0.0
```

Comment:
I really like what mig-operator did using `v99.0.0` for development. We should do the same.

References:
* https://github.com/konveyor/mig-operator/blob/c8f97a444bc32073a4b7e6d8f8f02df5a6de47c0/deploy/olm-catalog/bundle/manifests/crane-operator.v99.0.0.clusterserviceversion.yaml#L5 
* https://github.com/openshift/assisted-service/blob/364b9d71b11250ae536787ef354c2d7cb516bb7c/config/manifests/bases/assisted-service-operator.clusterserviceversion.yaml#L18